src/flb_network.c

```diff
@@ -841,8 +840,7 @@ static ares_socket_t flb_dns_ares_socket(int af, int type, int protocol, void *u
     lookup_context->ares_socket_type       = type;
     lookup_context->ares_socket_created    = FLB_TRUE;
 
-    lookup_context->response_event.mask    = MK_EVENT_EMPTY;
-    lookup_context->response_event.status  = MK_EVENT_NONE;
+    MK_EVENT_ZERO(&lookup_context->response_event);
```

Comment:
This MK_EVENT_ZERO should be placed just after malloc. It's in a different function.

Reply:
This was addressed in the review of PR 7001.